client/sites/components/sites-dataviews/actions.tsx

```diff
@@ -558,6 +563,17 @@ export function useActions( {
 				},
 				isEligible: isActionEligible( 'migrate-to-wpcom', capabilities ),
 			},
+
+			{
+				id: 'delete-site',
+				label: __( 'Delete site' ),
+				callback: ( sites ) => {
+					const site = sites[ 0 ];
+					page( `/sites/settings/administration/${ site.slug }/delete-site` );
```

Comment:
Let's use the existing URL (`/settings/delete-site/:slug`) for now, we have not released this URL yet.

Reply:
Updated.